src/orgasm_control.c

```diff
@@ -330,7 +334,13 @@ static void orgasm_control_updateEdgingTime() { // Edging+Orgasm timer
         // Detect if within post orgasm session
         if ((esp_timer_get_time() / 1000UL) < (post_orgasm_state.post_orgasm_start_millis +
                                                post_orgasm_state.post_orgasm_duration_millis)) {
-            output_state.motor_speed = Config.motor_max_speed;
+            if (output_state.motor_speed <= (Config.motor_max_speed - calculate_increment(
+                    Config.motor_start_speed, Config.motor_max_speed, Config.motor_ramp_time_s))
+                ) {
+                output_state.motor_speed += calculate_increment(
+                    Config.motor_start_speed, Config.motor_max_speed, Config.motor_ramp_time_s
```

Comment:
Is it possible that we can calculate this once at a higher scope and use that value down here for all these comparisons and increments? Speed isn't really a concern, but I think it would be tidy.

Reply:
true, i haven't looked that it becomes a "constant" unless start, max or ramp time is changed in config. I'll look into it tonight

Reply:
i just saw your update on motor control.
I had switched to  :        update_check(output_state.motor_speed, 0);

should i change all of these to your new fonction?  _set_speed(uint8_t speed)?

Reply:
Yes, use `_set_speed`, that will ensure both the hardware speed and the broadcasted events remain in sync.

`update_check` is used to set a flag if a value has changed, which is used by things like the UI and event manager to dispatch events. This should only be used for arousal and motor speed changes. I will likely clean this up as part of my efforts to simplify `orgasm_control.c`, since it is still inconsistent when `output_state.motor_speed` is used.